src/Network/HTTPRequestOptions.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Friendica\Network;
+
+use GuzzleHttp\RequestOptions;
+
+/**
+ * This class contains a list of possible HTTPClient request options.
+ */
+class HTTPRequestOptions
```

Comment:
According to the comment above, and to clear out confusion:
```suggestion
 * This class contains a list of possible HTTPClient request options.
 */
class HTTPClientOptions
```

Reply:
But in fact they're really request options (HTTPClient request options) ;-)

Reply:
OK, I changed it to `HTTPClientRequestOptions` :D

Reply:
Close, but not 1:1 so they are client options that we map to request options.

Reply:
hmm .. not fully convinced, but it's good for me :-)